grype/presenter/sarif/presenter_test.go

```diff
@@ -20,7 +20,7 @@ import (
 	"github.com/anchore/syft/syft/source"
 )
 
-var update = flag.Bool("update", true, "update the *.golden files for json presenters")
+var update = flag.Bool("update", false, "update the *.golden files for json presenters")
```

Comment:
great catch

Reply:
nit: since you're here it looks like the flag description could be updated  (it mentions updating the json presenter, but it's really for sarif)